migrations/2025-04-22-145900/up.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX ON tag (community_id);
```

Comment:
Check previous examples of create index, they're all named a certain way, starting with `idx_`

I checked the `post_tag` table, and its also missing a simple `post_id` index. Would probably be good to add one there also.

Reply:
added index name. the primary key in post_tag is already on post_id, so another index there would be redundant